.github/workflows/Playwright.yml

```diff
@@ -15,6 +15,19 @@ jobs:
         with:
           node-version-file: ".nvmrc"
 
+      - name: Set environment for branch
+        run: |
+          if [[ $GITHUB_REF_NAME == 'production' ]]; then
```

Comment:
should this be `[[ $GITHUB_REF_NAME == 'development' ]]`?

Reply:
since we talked about getting rid of the produciton/development branches and just using main for full CI/CD i'm wondering if this makes sense to use as a check. is there another check we could use?